lib/chewy/fields/base.rb

```diff
@@ -40,6 +40,8 @@ def mappings_hash
       def compose(*objects)
         result = evaluate(objects)
 
+        return {} if field_with_ignore_blank_flag?(result) || geo_point_field_without_ignore_blank_flag?(result)
```

Comment:
The thing here is that it would be nice to have a distinct default value for geo points (because blank values can't be indexed and raises an error).

But it would be also nice that, in case someone really wants to not ignore blanks and expect to have errors raised, I believe we should offer them this option.

Also remember that `''.blank?` and `[].blank?` returns true and we may want to have normal fields with empty strings or arrays indexed with those blank values.

Also notice that at method `geo_point_field_without_ignore_blank_flag?` she is checking `@options[:ignore_blank] != false` instead of `!@options[:ignore_blank]`, having default as `nil`, explicitly checking for not `false` handles this default without any extra effort